contributing/code/bugs.rst

```diff
@@ -32,9 +33,20 @@ If your problem definitely looks like a bug, report it using the official bug
 * Give as much detail as possible about your environment (OS, PHP version,
   Symfony version, enabled extensions, ...);
 
+* If you want to provide a stack trace you got on an html page, be sure to
+  provide the plain text version, which should appear at the bottom of the
+  page. *Do not* provide it as a screenshot, since search engines will not be
+  able to index the text inside them. Same goes for errors encountered in a
+  terminal, do not take a screenshot, but learn how to copy/paste from it. If
+  the stack trace is long, consider enclosing it in a `<details> html tag`_.
+  **Be wary that stack traces may contain sensitive information, and if it is
+  the case, be sure to redact them prior to posting your stack trace.**
```

Comment:
@alexislefebvre *edit* does not convey the full meaning, and I don't think we should dumb it down with remove. I think people can easily infer the meaning of the word if they don't know it already, and that means they will have learned something, so IMO it's better to stick with that.